### ISSUE TITLE
Feature Request: switch between km/l and l/100km

### ISSUE THREAD

**Issue by janleskovec** [akhilrex/hammond#105]
It would be nice to be able to switch the mileage chart to show fuel consumption in l/100km.

**Comment by bjinthahouse** [comment]
I wold like to second this! In most of Europe this is the unit of measurement for fuel consumption

**Comment by andrewtwelch** [comment]
Definitely something I think is worth adding, Australia uses l/100km so it would definitely be helpful for me!

**Comment by AlfHou** [comment]
Related #40

**Comment by alephtaph** [comment]
yes please, and the difference from the previous month's km to this month in the details page. So I see how far I came with a full tank.

**Comment by itmunky** [comment]
Came here to request the same thing. LPH / Litres per Hundred in Australia also
